Framework/include/QualityControl/TaskDataProcessorFactory.h

```diff
@@ -8,9 +8,12 @@
 
 #include "Framework/DataProcessorSpec.h"
 
-namespace o2 {
-namespace quality_control {
-namespace core {
+namespace o2
+{
+namespace quality_control
+{
+namespace core
+{
```

Comment:
How about changing nested namespaces to c++17 syntax - `namespace o2::quality_control::core {`? Not now, but when we have common agreement on that.

Reply:
I think that it is a good idea. I was not aware of it. Could you create a ticket in JIRA ? 
O2 General